RecoHI/HiTracking/python/hipixeltracks_miniAOD_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+hipixeltracks = cms.EDProducer("HIPixelTracksSlimmer_miniAOD",
+  srcTracks = cms.InputTag("hiConformalPixelTracks"),
+  srcVertices = cms.InputTag("offlinePrimaryVertices"),
+  srcPrimaryVertices = cms.InputTag("offlineSlimmedPrimaryVertices"),
+  dzSigCut = cms.double(10.0),
+  dxySigCut = cms.double(25.0),
+  dzSigHP = cms.double(7.0), #to set HighPurity flag for pixel tracks
```

Comment:
add these comments to the fillDescritptions

Reply:
This comment is included in this commit: https://github.com/cms-sw/cmssw/pull/30313/commits/deebf6766c730a8c97ac3f3fe90b065b24d5e70d